i18n/ja/docusaurus-plugin-content-docs/current/guides/integration-dsp-no-sdk.md

```diff
@@ -0,0 +1,123 @@
+
+- [Download a set of encryption keys](#downloadrefresh-encryption-keys)
+- [Decrypt UID2 Tokens into Raw UID2s](#decrypt-uid2-tokens-into-raw-uid2s)
+- [Periodically re-download the latest set of encryption keys](#downloadrefresh-encryption-keys)
+
+An example implementation, in the [UID2 SDK for C#&nbsp;/&nbsp;.NET](https://github.com/IABTechLab/uid2-client-net), is the `BidstreamClient` class: see [BidstreamClient.cs](https://github.com/IABTechLab/uid2-client-net/blob/6ac53b106301e431a4aada3cbfbb93f8164ff7be/src/UID2.Client/BidstreamClient.cs). This document refers to code sections from the C#&nbsp;/&nbsp;.NET SDK as examples of implementation.
+
+## Download/Refresh Encryption Keys
+
+To download or refresh the encryption keys, so that you can decrypt UID2 tokens into raw UID2s, call the following API endpoint:
+
+```
+/v2/key/bidstream
+```
+
+Refresh the keys once at startup, and then periodically (recommended refresh interval is hourly).
+
+The UID2 SDK for C#&nbsp;/&nbsp;.NET uses a `Refresh` function.
+
+To see how all the fields are parsed, refer to the UID2 SDK for C#&nbsp;/&nbsp;.NET `Parse` function: see [KeyParser.cs, lines 41-74](https://github.com/IABTechLab/uid2-client-net/blob/6ac53b106301e431a4aada3cbfbb93f8164ff7be/src/UID2.Client/KeyParser.cs#L41-L74).
+
+The API response is in JSON format, and includes `site_data`, the list of domains or app names that are allowed for the account.
+
+After decrypting the token into a raw UID2, you can use the information in `site_data` to verify that a specific domain or app name is on the list of names allowed for tokens that are generated on the client side. For details, see [Verify the Domain or App Name](#verify-the-domain-or-app-name).
+
+For more information about client-side UID2 implementation, refer to one of these implementation guides:
+
+- [Client-Side Integration Guide for JavaScript](publisher-client-side.md)
+- [Prebid.js](integration-prebid-client-side.md)
+- [Mobile](integration-mobile-client-side.md)
+
+To call the endpoint, you'll need to do two things:
+- Create an envelope for the request header.
+- Decrypt the response.
+
+For an implementation example, see [Encryption and Decryption Code Examples](../getting-started/gs-encryption-decryption.md#encryption-and-decryption-code-examples).
+
+## Decrypt UID2 Tokens into Raw UID2s
+
+When you have current keys you'll be able to decrypt a UID2 token into a raw UID2. You also need to check several conditions to make sure that the token is eligible for use in the bidstream.
+
+You'll need to complete the following steps:
+
+1. [Check the token version](#check-the-token-version).
+
+1. [Decrypt the token](#decrypt-the-token).
+
+1. [Make sure the token has not expired](#make-sure-the-token-has-not-expired).
+
+1. [Verify that the domain or app name is valid](#verify-the-domain-or-app-name).
+
+The UID2 SDK for C# / .NET uses a `DecryptTokenIntoRawUID` function to perform these steps.
```

Comment:
You can attach the link of `DecryptTokenIntoRawUID`
https://github.com/IABTechLab/uid2-client-net/blob/6ac53b106301e431a4aada3cbfbb93f8164ff7be/src/UID2.Client/BidstreamClient.cs#L15